### ISSUE TITLE
Train from scratch requirements 

### ISSUE THREAD

**Issue by Nadavc220** [YanchaoYang/FDA#15]
Hi,
Thanks for this contribution. 
What are the requirements for training the model from scratch? being more specific:
1) What gpu should be used?
2) how much gpu memory is needed?
3) which pytorch version?
4) which python version?
5) which cuda version?

I there anything crucial needed for reproducing the results of the paper? from my experience minor changes in the training environment can make a big difference.

Thanks =)

**Comment by victorvini08** [comment]
Heyy mann!....We had the same doubts like yours especially in the part of generation of pseudo labels. We are using a tesla T4 gpu but the get pseudo label script is getting killed after processing half images.....Could you find any solution??

**Comment by Nadavc220** [comment]
Hi,
Unfortunately, I did not get to the point of running the pseudo label script yet. I guess I will get there eventually and will try o tackle your problem.
If you get to solve this I will be happy if you can find the time to post it. I will do the same if I get there before you.

**Comment by KaiiZhang** [comment]
> Heyy mann!....We had the same doubts like yours especially in the part of generation of pseudo labels. We are using a tesla T4 gpu but the get pseudo label script is getting killed after processing half images.....Could you find any solution??

It is caused by insufficient memory. Because all prediction results of the entire dataset need to be stored in memory for sorting. In my implementation, I used 97G of memory.
